### PR TITLE
Feature/kieslerkreis page

### LINKED ISSUE #46: add Kieslerkreis OverviewPage
## Value proposition

As an User
I want to have a page with all my Kieslerkreis-Results
So that I can see my behavior, my wanted behavior and more information about the kieslerkreis

## Description

Image

Image

Image

## Acceptance criteria

- [ ] have a `new page` where i see the kieslerkreis
- [ ] have the results from the entries mapped on the kieslerkreis
- [ ] take the results from `behaviorKieslerkreis` and `behaviorChangedKieslerkreis`
- [ ] have a `description` about the kieslerkreis to the bottom

### PR DESCRIPTION
**Pull request by vaupunkt**
# PR for [UserStory](https://github.com/vaupunkt/cbaspCompanion/issues/46)

As an User
I want to have a page with all my Kieslerkreis-Results
So that I can see my behavior, my wanted behavior and more information about the kieslerkreis

### CODE REVIEW

File: components/AnalysisForm/index.js [hunk 1]
```diff
@@ -247,8 +237,7 @@ export default function AnalysisForm({
               <KieslerKreis
                 editMode="true"
                 analysisKey={analysisKey}
-                kieslerkreisData={kieslerkreisData}
-                setKieslerkreisData={setKieslerkreisData}
+                kieslerkreisDataset="[null,null,null,null,null,null,null,null]"
```
Comment (josch87) [inline components/AnalysisForm/index.js]: Why do you need an array with so many NULLs? Is there a simpler/nicer way to handle null on the KieslerKreis component when kieslerkreisDataset is falsy? Maybe a default value for the prop with an (empty?) array?

File: components/KieslerKreis/index.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import { Radar } from "react-chartjs-2";
 import Chart from "chart.js/auto";
```
Comment (josch87) [inline components/KieslerKreis/index.js:2]: You don't need that import anymore.

File: pages/kieslerkreis/index.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    return result;
+  }
+  const overviewDataset = {
+    averageBehaviorKieslerkreis: averageArrays(behaviorKieslerkreisArray),
+    averageBehaviorChangeKieslerkreis: averageArrays(
+      behaviorChangeKieslerkreisArray
+    ),
+  };
+  console.log(overviewDataset);
```
Comment (josch87) [inline pages/kieslerkreis/index.js:46]: There is a console.log left.

Comment (josch87) [reply pages/kieslerkreis/index.js:46]: Also have a look at the console.log in newanalysis/index.js line 96.

**Review by josch87 — CHANGES_REQUESTED** [review]
Hi. Looks very good, just some minor comments.

File: pages/kieslerkreis/index.js (hunk 3 above)
Comment (vaupunkt) [reply pages/kieslerkreis/index.js:46]: thanks! 👀

File: components/KieslerKreis/index.js (hunk 2 above)
Comment (vaupunkt) [reply components/KieslerKreis/index.js:2]: actually even if it seems unused it is needed for react-chartjs-2 to work 😊

File: components/AnalysisForm/index.js (hunk 1 above)
Comment (vaupunkt) [reply components/AnalysisForm/index.js]: i just noticed i already implemented a workaround at the useState function in the kieslerkreis-component. so i can just delete this line 🙏🏻

**Review by tejue — APPROVED** [review]
This code is impressive, Veliko!

**Pull request merged**